test/integration/scheduler/scheduler_test.go

```diff
@@ -107,7 +107,8 @@ func TestSchedulerCreationFromConfigMap(t *testing.T) {
 			"priorities" : [
 				{"name" : "PriorityOne", "weight" : 1},
 				{"name" : "PriorityTwo", "weight" : 5}
-			]
+			],
+			"hardPodAffinitySymmetricWeight" : 10
 			}`,
```

Comment:
You also need to verify the config is applied as others, such as `schedPrioritizers` etc.